rust/private/rust.bzl

```diff
@@ -685,6 +685,7 @@ _common_attrs = {
         default = Label("//util/import"),
         cfg = "exec",
     ),
+    "_is_proc_macro_dep": attr.label(default = Label("//:is_proc_macro_dep")),
```

Comment:
nit: make this consistent with the lines above

Reply:
Thanks! Updated.